scripts/linters/excluded_any_type_files.json

```diff
@@ -0,0 +1,161 @@
+[
```

Comment:
I made a separate file for the exceptions because they are large in number and writing these in the code itself, didn't seem right.